platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/CMakeLists.txt

```diff
@@ -26,6 +34,37 @@ target_sources(mbed-psa
         pal/pal_internal_trusted_storage_intf.c
         pal/pal_protected_storage_intf.c
 
+        services/attestation/attest_boot_status_loader.c
+        services/attestation/attest_crypto.c
+        services/attestation/attest_crypto_keys.c
+        services/attestation/attest_iat_claims_loader.c
+        services/attestation/attestation_bootloader_data.c
+        services/attestation/psa_attest_inject_key.c
+        services/attestation/psa_attestation_stubs.c
+        services/attestation/psa_initial_attestation_api.c
+        services/attestation/psa_inject_attestation_key_impl.c
+
+        services/attestation/qcbor/src/UsefulBuf.c
+        services/attestation/qcbor/src/ieee754.c
+        services/attestation/qcbor/src/qcbor_decode.c
+        services/attestation/qcbor/src/qcbor_encode.c
+
+        services/attestation/tfm_impl/attest_token.c
+        services/attestation/tfm_impl/attestation_core.c
+
+        services/attestation/tfm_impl/t_cose/src/t_cose_sign1_sign.c
+        services/attestation/tfm_impl/t_cose/src/t_cose_util.c
+
+        services/platform/platform_emul.c
+        services/platform/platform_srv_impl.c
+
+        services/storage/common/psa_storage_common_impl.cpp
+
+        services/storage/its/pits_impl.cpp
+        services/storage/ts/psa_prot_internal_storage.cpp
```

Comment:
an error comes from here, `its`, I'll fix and restart

Reply:
Which error?